### ISSUE TITLE
Client Feedback 

### ISSUE THREAD

**Issue by adelaiderobinson** [iMPAct-capstone/WildAid_data_entry#12]
- [x] automatically scroll to the top when you switch to a new tab 
- [x] loading icon 
- [ ] warning if the user tries to exit the page without saving? 
- [x] way of letting users know how much of the tracker has/hasn't been filled out/which fields are missing
- [x] add numbers to categories
- [x] collapse and expand boxes as you go through a tab (low priority)
- [x] try to be able to view previous scores as the user fills out the current tracker
- [x] respond to Silvia's comment about the tracker being able to be in Spanish

**Comment by adelaiderobinson** [comment]
collapsed score guidelines instead of entire box

**Comment by adelaiderobinson** [comment]
got loading bar working for first page and can add to other tabs but the problem is that that the loading bar won't show up if the app is waiting on another user, which is really what will slow it down-not sure what to do about this issue

**Comment by adelaiderobinson** [comment]
Marking as closed due to end of capstone project, moved remaining task to possible future updates issues.
